lighthouse-core/test/gather/gatherers/full-page-screenshot-test.js

```diff
@@ -20,7 +19,7 @@ function createMockDriver({contentSize, screenSize, screenshotData}) {
         return contentSize.width;
       }
       if (code === 'window.devicePixelRatio') {
-        return screenSize ? screenSize.dpr : 1;
+        return screenSize ? screenSize.dpr : 2;
```

Comment:
i don't understand this change

Reply:
good call.  it's kind of a separate change.  This is the mock driver so we're returning a DPR back to lighthouse.

Since we're now emulating DPR 1 to take the screenshot.. I figured I'd just pretend the browser is a non-1 DPR just to keep things separate

afaik this value is never even used.......

......

okay in that case i'll just delete this part of the mock.
done.